### PR TITLE
Minor enhancements and tweaks

### LINKED ISSUE #455: Inconsistent parameter headers
**Describe the bug**
A recent fix to the parameter headers changed the _1, _2, _SN, and _CP identifier suffixes to (1), etc. However, this was not applied consistently to all parameters.

In the CEs output, I see the following parameters:

['BE_Fixed(1)', 'BE_Fixed(2)', 'BE_Kruckow(1)', 'BE_Kruckow(2)', 'BE_Loveridge(1)', 'BE_Loveridge(2)', 'BE_Loveridge_Winds(1)', 'BE_Loveridge_Winds(2)', 'BE_Nanjing(1)', 'BE_Nanjing(2)', 'Binding_Energy<CE(1)', 'Binding_Energy<CE(2)', 'CE_Event_Count', 'Core_Mass_1', 'Core_Mass_2', 'Double_Core_CE', 'Eccentricity<CE', 'Eccentricity>CE', 'Immediate_RLOF>CE', 'Lambda@CE(1)', 'Lambda@CE(2)', 'Lambda_Fixed(1)', 'Lambda_Fixed(2)', 'Lambda_Kruckow(1)', 'Lambda_Kruckow(2)', 'Lambda_Lambda_Nanjing(1)', 'Lambda_Lambda_Nanjing(2)', 'Lambda_Loveridge(1)', 'Lambda_Loveridge(2)', 'Lambda_Loveridge_Winds(1)', 'Lambda_Loveridge_Winds(2)', 'Luminosity<CE(1)', 'Luminosity<CE(2)', 'MT_History', 'Mass_1<CE', 'Mass_2<CE', 'Mass_Env_1', 'Mass_Env_2', 'Merger', 'Optimistic_CE', 'RLOF(1)', 'RLOF(2)', 'Radius_1<CE', 'Radius_1>CE', 'Radius_2<CE', 'Radius_2>CE', 'RocheLobe_1<CE', 'RocheLobe_1>CE', 'RocheLobe_2<CE', 'RocheLobe_2>CE', 'SEED', 'SemiMajorAxis<CE', 'SemiMajorAxis>CE', 'Simultaneous_RLOF', 'Stellar_Type(1)', 'Stellar_Type(2)', 'Stellar_Type_1<CE', 'Stellar_Type_2<CE', 'Tau_Circ', 'Tau_Dynamical<CE(1)', 'Tau_Dynamical<CE(2)', 'Tau_Nuclear<CE(1)', 'Tau_Nuclear<CE(2)', 'Tau_Radial<CE(1)', 'Tau_Radial<CE(2)', 'Tau_Sync', 'Tau_Thermal<CE(1)', 'Tau_Thermal<CE(2)', 'Teff<CE(1)', 'Teff<CE(2)', 'Time', 'Zeta_Lobe', 'Zeta_Star']

Some of the parameters have the new suffix style, but some keep the old version. I thought this might have to do with whether the _X characters were at the end of the parameter name, but there are some which do not follow that rule either, e.g Core_Mass_1.

**Label the issue**
`urgency_low`      - This issue is not urgent
`severity_minor`    - This is a minor bug with minimal impact

**To Reproduce**
Run Compas with default settings, look at CEs output header.

**Expected behavior**
Parameter names should be consistent.

**Versioning (please complete the following information):**
 - COMPAS [e.g. v02.17.03]

### PR DESCRIPTION
**Pull request by jeffriley**
(1) Added orbital-period-distribution option (see note in Options.cpp re orbital period option)
(2) Added mass-ratio option
(3) Updated default pythonSubmit to reflect new options, plus some previous omissions (by me...)
(4) Minor typo/formatting changes throughout
(5) Updated docs for new options, plus some typos/fixes/previous omissions

Docs changed for issue #455
Docs changed for PR #458

### CODE REVIEW

**Comment by jeffriley** [comment]
Once this PR is merged, COMPAS will provide options to:

- specify an explicit value
- sample a value, and specify the range of values that should be sampled (i.e. minimum and maximum), and, where COMPAS provides more than one distribution, specify which distribution (but see below)

for all of the following initial attributes of a star/binary:

- initial mass (single star for SSE; primary, and secondary for BSE)
- metallicity
- eccentricity
- semi-major axis
- orbital period (semi-major axis takes precedence if specified)
- mass ratio (m2/m1)

(for the following, just explicit values, or uniformly sampled random values - so can't specify distribution)

- kick magnitude (single star for SSE; primary, and secondary for BSE)
- kick magnitude random number (single star for SSE; primary, and secondary for BSE)
- kick theta (primary, and secondary for BSE; not defined for SSE)
- kick phi (primary, and secondary for BSE; not defined for SSE)
- kick mean anomaly (primary, and secondary for BSE; not defined for SSE)

COMPAS C++ code currently provides at least one distribution for sampling each of the initial attributes.  The plan is for the COMPAS C++ code to provide a single distribution for each, so that users can use COMPAS straight out of the box, and either specify an explicit value for each initial attribute or let COMPAS sample a value (from the 'convenience' distribution we provide).  If users want to use different distributions then they will use Stroopwafel, or write their own scripts, to construct grid files.  That way we only have to code and maintain a single distribution for each of the initial attributes - users will be responsible for sampling from their personal favourite distributions.

(So at some stage we should decide on which single distribution for each of the initial attributes to keep in the COMPAS C++ code and remove the rest).

**Review by ghost — APPROVED** [review]
Looks great, thank you, @jeffriley !

**Pull request merged**
